modules/upgrade/pages/deprecated/index.adoc

```diff
@@ -249,6 +250,20 @@ Prior to `rpk` supporting the `-X` flag, each common configuration option was it
 
 ====
 
+// This feature is not supported in cloud
+ifndef::env-cloud[]
+== Kubernetes
+
+[.sortable]
+|===
+| Deprecated in  | Feature | Details
+
+| Operator v2.2.3-24.2.X
+| Migration path from the deprecated Cluster and Console custom resources.
+| As part of our ongoing enhancements, the deprecated Cluster and Console custom resources will not be migrated to the new Flux-based version. In the future, these custom resources will leverage the same reconciler that is currently used by Redpanda, ensuring a consistent and unified experience across all resources.
```

Comment:
I don't think we should document the intent to unify the Cluster resource. That initiative is aimed at helping cloud and we don't want to encourage self hosted users to start using the Cluster CRD. 

...We don't document anything about the Cluster or Console CRD for end users aside from this migration, right?

Reply:
Thanks, Chris. Updated.